### ISSUE TITLE
Option to specify the report name

### ISSUE THREAD

**Issue by embee1981** [SpacialCircumstances/gradle-cucumber-reporting#8]
Would it be possible to expose an option to specify the report name in the config?

I'm seeing "root project 'myapp'" and I dunno where it has got the 'root project' aspect from?

The only reference to the project name I have is in settings.gradle

rootProject.name = 'myapp'

**Comment by SpacialCircumstances** [comment]
I've released version 0.1.8, which should allow you to do this now. If this property is not specified, it will use your gradle project name instead.

```
cucumberReports {
    outputDir = file("$projectDir/output/generated-reports")
    buildId = '0'
    reports = files("$projectDir/reports/example-report.json")
    projectNameOverride = "YOUR PROJECT NAME HERE"
}
```

**Comment by embee1981** [comment]
Thanks for the quick response and fix 😊 Works a charm
